recipes/clermontyping/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "1.0.2" %}
-{% set sha256 = "6b14a7a1450f13dfc5fc4ca39349baf94d9a09b726f982213119284d06bf2f8c" %}
+{% set version = "24.02" %}
```

Comment:
@happykhan Is this OK ? The version jumps from 1.x to 24.x.
If you don't use SemVer then please change
`{{ pin_subpackage('clermontyping', max_pin="x") }}  `
to 
`{{ pin_subpackage('clermontyping', max_pin=None) }}  `

Reply:
Fine with me;